TLM/TMPE.API/Traffic/Enums/SimulationAccuracy.cs

```diff
@@ -4,6 +4,7 @@ public enum SimulationAccuracy {
         Low = 1,
         Medium = 2,
         High = 3,
-        VeryHigh = 4
+        VeryHigh = 4,
+        MaxValue = 4
```

Comment:
Add comment here explaining purpose of `MaxValue`?

Reply:
my IDE complains there should be `,` even after the final element. just a note its your choice.

Reply:
My IDE complains about redundant comma if I add it, lol

Reply:
as i said in the chat you could also consider using something like this in `shortcuts.cs`:
```
        public static T GetMaxEnumValue<T>()
            where T: Enum=>
            Enum.GetValues(typeof(T)).Cast<T>().Max();
```

Reply:
I don't think we would need that. Associated enum value is not used used to calculate anything just for enhance readability of comparisons and `MaxValue` clearly describes what it does.